src/v2/view-builder/views/google-authenticator/EnrollAuthenticatorGoogleAuthenticatorView.js

```diff
@@ -0,0 +1,126 @@
+import { loc, View, createButton } from 'okta';
+import hbs from 'handlebars-inline-precompile';
+import BaseForm from '../../internals/BaseForm';
+import BaseView from '../../internals/BaseView';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import AuthenticatorEnrollFooter from '../../components/AuthenticatorEnrollFooter';
+import EnrollGoogleAuthenticatorBarcodeView from './EnrollGoogleAuthenticatorBarcodeView';
+import EnrollAuthenticatorManualSetupView from './EnrollAuthenticatorManualSetupView';
+
+const VIEW_TO_DISPLAY = 'viewToDisplay';
+const viewToDisplayState = {
+  BARCODE: 'barcode',
+  MANUAL: 'manual',
+  ENTER_CODE: 'enterCode',
+};
+
+const Body = BaseForm.extend({
+  title () {
+    return loc('oie.enroll.google_authenticator.setup.title', 'login');
+  },
+
+  noButtonBar: true,
+
+  className: 'oie-enroll-google-authenticator',
+
+  getUISchema () {
+    const schema = BaseForm.prototype.getUISchema.apply(this, arguments);
+
+    schema[0].showWhen = {
+      viewToDisplay: viewToDisplayState.ENTER_CODE,
+    };
+
+    const enterCodeSubtitle = View.extend({
+      template: hbs`
+      <div class="google-authenticator-setup-info-title enter-code-title">
+      {{i18n code="oie.enroll.google_authenticator.enterCode.title" bundle="login"}}
+      </div>
+    `,
+    });
+
+    const nextButton = createButton({
+      attributes: {
+        style: 'display: block'
+      },
+      className: 'google-authenticator-next button-primary default-custom-button',
```

Comment:
I see at many places in the code we are doing this `button-primary default-custom-button`, this needs to be generalized as well.